artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/config/PersistedBridgeConfiguration.java

```diff
@@ -62,7 +62,7 @@ public void decode(ActiveMQBuffer buffer) {
    }
 
    public String getName() {
-      return bridgeConfiguration.getParentName();
+      return bridgeConfiguration.getName();
```

Comment:
is this a required change? it seems un-related.

Reply:
@clebertsuconic It's not strictly necessary, but without it each configuration reload will log WARN messages (AMQ222124).

I'll go ahead and change it so that the `ActiveMQServerImpl#recoverStoredBridges()` fetches this value from the underlying bridgeConfig instead and leave the `PersistedBridgeConfiguration` as it was.

Reply:
If needed keep it.. just trying to understand why and if it was not a left over from a debug or something.

Reply:
@AntonRoskvist I wasn't trying to prevent a change to stating it was wrong BTW.